lib/barcelona/network/auto_scaling_group.rb

```diff
@@ -11,7 +11,7 @@ def define_resource(json)
           j.DesiredCapacity desired_capacity
           j.Cooldown 0
           j.HealthCheckGracePeriod 0
-          j.MaxSize(desired_capacity + 1)
+          j.MaxSize(desired_capacity * 2 + 1)
```

Comment:
Sorry that was an answer to "why `+1` is needed". Here's why `* 2` is needed:
When rolling update runs, it doesn't change MaxSize of the autoscaling. What I want do in this PR is to replace all running instances at the same time so running instance count changes as follows

N (count before update) -> N*2 (old instances + new instances) -> N (new instances)

If `* 2` is not added, the above flow doesn't work because `MaxSize` is N + 1 and CloudFormation adds instances one by one